hardware/raspberrypi/bootmodes/host.md

```diff
@@ -1,4 +1,6 @@
-# USB Host boot mode
+# USB host boot mode
+
+**USB host boot is available on the Raspberry Pi 3B, 3B+, and Raspberry Pi 2B v1.2 models only. The 3A+ supports USB host boot as well, but only for mass storage device boot, not network boot.*
```

Comment:
Perhaps this would sound better as "USB host boot is available on the Raspberry Pi 3B, 3B+, 3A+ and Raspberry Pi 2B v1.2 models only. The 3A+ only supports mass storage boot, not network boot." ?

Reply:
I agree - that's probably what the copy editors would have changed it to 👍.